torchbenchmark/models/speech_transformer/__init__.py

```diff
@@ -59,15 +59,15 @@ def set_module(self, new_model):
         elif self.test == "eval":
             self.evalcfg.model = new_model
 
-    def train(self, niter=1):
+    def _train(self, niter=1):
```

Comment:
nit: maybe there's some way to avoid renaming the method for every single model? You could perhaps do some `invoke_train` wrapper or move the wrapping to a different place

Reply:
I like the idea! Will make this change.

Reply:
Updated: now all tests have the same entry `invoke()`, and we dispatch to either `train()` or `eval()` by the value of `self.test`.